src/vm_pages.rs

```diff
@@ -68,9 +69,11 @@ pub type InstructionFetchResult = core::result::Result<DecodedInstruction, Instr
 /// The number of pages for the `VmRegionList` vector.
 pub const TVM_REGION_LIST_PAGES: u64 = 1;
 
-/// The base number of state pages required to be donated for creating a new VM. For now, we just need
-/// one page to hold the VM state itself and whatever is required to hold the `VmRegionList`.
-pub const TVM_STATE_PAGES: u64 = 1 + TVM_REGION_LIST_PAGES;
+/// The number of pages to hold `Vm` state.
+pub const TVM_VM_STATE_PAGES: u64 = 3;
```

Comment:
Could we avoid hard-coding this constant with something like this?

```
pub const TVM_VM_STATE_PAGES: u64 = PageSize::num_4k_pages(core::mem::size_of::<Vm>());
```

Reply:
Yes.